### PR TITLE
Proofreading and adding missing translation

### PR DESCRIPTION
**Pull request by gaelcoral**

### CODE REVIEW

File: resource/gameui_latam.txt [hunk 1]
```diff
@@ -1075,8 +1075,8 @@
 "GameUI_Icons_DOWNCURSOR"		">"
 "[english]GameUI_Icons_NONE"		"< not assigned >"
 "GameUI_Icons_NONE"		"< sin asignar >"
-		"[english]GameUI_KeyNames_Not_Bound"		"< not bound >"
-		"GameUI_KeyNames_Not_Bound"		"< not bound >"
+"[english]GameUI_KeyNames_Not_Bound"		"< not bound >"
+"GameUI_KeyNames_Not_Bound"		"< no vinculado >"
```
Comment (Blueberryy) [inline resource/gameui_latam.txt]: Can you shorten this text? It might not fit on instructor tip.

**Comment by Blueberryy** [comment]
Should i merge your PR right now?

**Comment by gaelcoral** [comment]
> Should i merge your PR right now?

I would say not yet, because I am still checking which strings fit and which do not.

**Comment by gaelcoral** [comment]
> I would say not yet, because I am still checking which strings fit and which do not.

Hey, now you can merge my pull request, I have fixed some strings that didn't fit.
All this PR files are done.
image

**Comment by Blueberryy** [comment]
> > I would say not yet, because I am still checking which strings fit and which do not.
> 
> Hey, now you can merge my pull request, I have fixed some strings that didn't fit. All this PR files are done. image

Now i hope that Latam translation will be available in next update. Yay

**Pull request merged**